File: 2.0/java/contacts/updateContact.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.contacts.model.Contact;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        Contact contact = new Contact();
        contact.setId(11646003L);
        contact.setMobilePhone("12136600123");
        contact.setWorkPhone("14553320089");
        client.contactsApi().update(contact);
    }
}

File: 2.0/csharp/contacts/updateContact.cs
using CallfireApiClient;
using CallfireApiClient.Api.Contacts.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var contact = new Contact
        {
            Id = 11646003,
            MobilePhone = "12136600123",
            WorkPhone = "14553320089"
        };
        client.ContactsApi.Update(contact);
    }
}
